similarity-data-impl/src/main/resources/ApplicationResources.properties

```diff
@@ -1,16 +1,73 @@
+phenotips.similarCases.title=Similar cases
+phenotips.similarCases.subtitle=Matches with patient records in this database
+
 phenotips.similarCases.details.show=Show phenotype and genotype similarity...
 phenotips.similarCases.details.hide=Hide phenotype and genotype similarity...
-
 phenotips.similarCases.currentPatient.title=The current patient
 phenotips.similarCases.matchedPatient.title=The matched patient
-
 phenotips.similarCases.ageOfOnset=Age of onset
-
 phenotips.similarCases.modeOfInheritance=Mode of inheritance
-
 phenotips.similarCases.phenotypicFeaturesBreakdown=Phenotypic features breakdown
-
 phenotips.similarCases.geneMatchingBreakdown=Gene matching breakdown
-
 phenotips.similarCases.showDisclaimer=Show disclaimer
 phenotips.similarCases.disclaimerDialogTitle=Service Disclaimer
+phenotips.similarCases._UNKNOWN_INFO_MARKER=unknown
```

Comment:
Why name these 4 strings differently form all others, in capitals with an underscore?

Reply:
no particular reason, I guess